zoos/zoos-router.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express")
+const knex = require("knex")
+const knexConfig = require("../knexfile")
+
+const db = knex(knexConfig.development)
+const router = express.Router()
+
+router.post("/", (req, res) => {
+  const name = req.body.name
+  if (!name) {
```

Comment:
You can also use a destructuring assignment if you want to use es6 syntax. `const { name } = req.body`